borrowing/views.py

```diff
@@ -1,3 +1,74 @@
-from django.shortcuts import render
+from rest_framework import generics, status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from borrowing.models import Borrowing, Book
+from borrowing.serializers import BorrowingSerializer
+from django.utils import timezone
 
-# Create your views here.
+
+class BorrowingListCreateAPIView(generics.ListCreateAPIView):
+    queryset = Borrowing.objects.all()
+    serializer_class = BorrowingSerializer
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        user_id = self.request.query_params.get("user_id")
+        is_active = self.request.query_params.get("is_active")
+
+        if user_id:
+            queryset = queryset.filter(user_id=user_id)
+
+        if is_active:

```

Comment:
we have to add (if is_active is not None: .... )
is_active can be False like filter and then we won't pass this check

Reply:
or just add one more check for False